i18n/en.json

```diff
@@ -4044,7 +4048,9 @@
   "upload_overlay.info": "Drop a file to upload it.",
   "user_list.notFound": "No users found",
   "user_profile.account.editSettings": "Edit Account Settings",
-  "user_profile.account.localTime": "Local Time: ",
+  "user_profile.account.localTime": "Local Time",
+  "user_profile.custom-status": "Status",
+  "user_profile.custom-status.set-status": "Set a status",
```

Comment:
Minor nitpicking, but this should either be `user_profile.customStatus` or `user_profile.custom_status` without the hyphens. Not that we're very consistent with these IDs, but I think we don't use hyphens for them at least.